cardano-db/src/Cardano/Db/Insert.hs

```diff
@@ -134,7 +134,7 @@ insertEpochSyncTime :: (MonadBaseControl IO m, MonadIO m) => EpochSyncTime -> Re
 insertEpochSyncTime = insertReplace "EpochSyncTime"
 
 insertExtraKeyWitness :: (MonadBaseControl IO m, MonadIO m) => ExtraKeyWitness -> ReaderT SqlBackend m ExtraKeyWitnessId
-insertExtraKeyWitness = insertCheckUnique "ExtraKeyWitness"
+insertExtraKeyWitness = insertUnchecked "ExtraKeyWitness"
```

Comment:
If we remove the `txId` from the `UniqueWitness` constraint, it is probably unnecessary to use `insertUnchecked`.